scripts.js

```diff
@@ -0,0 +1,123 @@
+// Wait until the document is ready
+$(document).ready(function(){
+
+  // URL of the API
+  var url = 'https://trektravel.herokuapp.com/trips';
+
+  // Show the list of vacations
+  var successCallback = function (response) {
+    for (var i=0; i < response.length; i++ ){
+      $('#vacations').append("<ul><li><a href=" + url + "/" + response[i].id + ">" + response[i].name + "</a></li><li> Continent:" + response[i].continent + "</li><li> Weeks:" + response[i].weeks + "</li></ul>");
+    } // end for loop
+  }; // end successCallbank function
+
+  // Click to show the list, then hide the button;
+  $('#show_vac').on('click', function(e) {
+    $.get(url, successCallback);
+    $("#show_vac").hide();
+  }); // end onclick function
+
+  // declare a variable (unassigned to start)
+  var openStatus;
+  console.log(openStatus);
+
+  // Start of vacations on link click function
+  $('#vacations').on('click', 'a', function(e) {
+    // stop the click on a vacation link from going to url
+    e.preventDefault();
+
+    // change display status div single-vac to block
+    $('#single_vac').show();
+
+    // set local URL
+    var vacUrl = $(this).attr('href');
+
+    // make get request for all the details of a single vacation
+    $.get(vacUrl, function(vacation){
+      $('#name').text("Vacation Name: " + vacation.name);
+      $('#destination').text("Destination: " + vacation.destination);
```

Comment:
In the trips I checked, the destination was `undefined`, you may need to check here.  

I think the field you're looking for is:  continent.